README.md

```diff
@@ -4,27 +4,30 @@ This application migrates LTI tools in a Canvas instance from one version to ano
 
 ## Running with Docker compose
 
-This is the recommended way to run the application.  Docker compose will ensure the application is run in a consistent environment with the correct version of Python and all the required dependencies.
+Using Docker compose is the recommended way to run this application.  Docker compose will ensure the application is run in a consistent environment with the correct version of Python and all the required dependencies.
 
 ### Prerequisites
 * [Docker Desktop](https://www.docker.com/products/docker-desktop/) — This must be installed first.
-* [tool-migration](https://github.com/tl-its-umich-edu/tool-migration/releases/latest) — Download the latest release of this app and unzip it.  The latest release will be at the top of the list.  The zip file is found under the heading "Assets".  (Alternatively, the repository can be cloned.)
+* [tool-migration](https://github.com/tl-its-umich-edu/tool-migration/releases/latest) — Download the latest release of this app and unzip it.  The latest release will be at the top of the list.  The zip file is found under the heading "Assets".  (Alternatively, its repository can be cloned.)
 
 ### Configuration
 
-In a shell (e.g., using Terminal.app on macOS), navigate to the directory where the app was unzipped or cloned.  Then copy the sample configuration file:
+In a shell (e.g., using Terminal.app on macOS), navigate to the directory where the app was unzipped or cloned.  Then copy the sample configuration file to the file that will be used by the application, `env`:
 
 ```shell
-cp .env.sample .env
+cp env.sample env
 ```
 
-Then open the newly copied `.env` file with an editor and fill in the missing values.  E.g., to edit with TextEdit.app on macOS:
+Then open the newly copied `env` file with an editor and fill in the missing values.  E.g., to edit with TextEdit.app on macOS:
 
 ```shell
-open -a TextEdit .env
+open -a TextEdit env
 ```
 
-Note: Starting the editor this way is recommended because the hidden file `.env` is not visible in the editor's "Open" dialog.
+Carefully fill in the values for each of the variables.  The values must be surrounded by quotes.  E.g.,
+
+```shell
```

Comment:
Should a sample variable setting be listed here?

Reply:
That's not a bad idea.